core/controllers/profile.py

```diff
@@ -264,6 +264,8 @@ def post(self):
             except utils.ValidationError as e:
                 raise self.InvalidInputException(e)
 
+        user_services.set_signup_profile_picture_url(self.user_id)
```

Comment:
To make sure we store the profile picture only at the signup , we call set_signup_profile_picture_url method defined in user_services module